runtime/src/main/java/io/quarkiverse/githubapp/runtime/config/ConfigChecker.java

```diff
@@ -0,0 +1,48 @@
+package io.quarkiverse.githubapp.runtime.config;
+
+import java.util.Set;
+import java.util.TreeSet;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+import io.quarkus.runtime.LaunchMode;
+
+@Singleton
+public class ConfigChecker {
+
+    @Inject
+    GitHubAppRuntimeConfig gitHubAppRuntimeConfig;
+
+    @Inject
+    LaunchMode launchMode;
+
+    public void checkConfig() {
+        Set<String> missingPropertyKeys = new TreeSet<>();
+
+        if (gitHubAppRuntimeConfig.appId.isEmpty()) {
+            missingPropertyKeys.add("quarkus.github-app.app-id");
+        }
+        if (gitHubAppRuntimeConfig.privateKey.isEmpty()) {
+            missingPropertyKeys.add("quarkus.github-app.private-key");
+        }
+        if (launchMode == LaunchMode.NORMAL && gitHubAppRuntimeConfig.webhookSecret.isEmpty()) {
+            missingPropertyKeys.add("quarkus.github-app.webhook-secret");
+        }
```

Comment:
I was doing this in the constructor on purpose, so that `checkConfig` could be called very often without any problem.